### ISSUE TITLE
Standardize file names of DocBook/ADoc files

### ISSUE THREAD

**Issue by ghost** [SUSE/doc-styleguide#15]
In doc-sle, it can be hard to see which element belongs to which book. Finding out for sure can be a lot of `grep`ping/`daps list-srcfile`ing. Therefore, we should have a standardized file name structure for XML and DC files.

My proposal would be, since this would require quite a bit of renaming, to start on this with 12 SP3.

Proposal for DC files:
- prefix: `DC-`
- acronym of product: `SLES` (for SUSE Linux Enterprise Server)
- hyphen: `-`
- most relevant word (usually, noun in singular) from the book name: `security` (for Security Guide)

Proposal for XML main files:
- prefix: `MAIN.`
- acronym of product/product group: `SLE` (for SLES)
- file extension: `.xml`

Proposal for XML files:
- most relevant word (usually, noun in singular) from the book name: `security` (for Security Guide), if included in multiple books/articles: `common`
- dot: `.`
- abbreviation of type: `book`, `art`, `cha`, `sec`, or `_i` (for second-level includes) 
- sections/chapters/second-level includes only: most relevant word(s) (usually, noun in singular) from the chapter/section/include name: `kerberos` (for Kerberos section), multiple words are separated by hyphens:  `certificate-store` (for Certificate Store section)
- file extension: `.xml`

**Comment by sknorr** [comment]
@taroth21 @asettle @lproven 

Notably, there are a few areas that are underdefined:

* In doc-sle, where we have both sets and the books they contain referenced from the MAIN parameter of DC files. Should sets and books and articles all be named `MAIN. ... .xml`?
* What should main files look like otherwise? Do they need to include a `set-`/`book-`/`art-` prefix?
* What should book and article files that are not MAIN files be named? E.g. `security.book.xml` or `security.book-security.xml`

Otherwise, does this spark joy?

![Screenshot from 2020-08-20 18-16-24](https://user-images.githubusercontent.com/5476547/90798500-4b45bf00-e312-11ea-9ad6-8e943d9ee962.png)

**Comment by asettle** [comment]
This formatting standard totally affects SOC too, it's a mess. This is a good idea to implement and roll on forwards, especially considering how much bigger SES is getting with the new content for Rook + cephadm. 

In favour of this change being made a permanent one across all product docs.

**Comment by sknorr** [comment]
As cwickert just noted on the call, images should also have some kind of schema but let's not discuss here & open a new issue.

**Comment by jfaltenbacher** [comment]
I am always in favor of standardization :) the question is, if the scheme we have agreed on for the smartdocs would be enough and if we want to change our current xmls - that would result in extra workload

**Comment by janajaeger** [comment]
> I am always in favor of standardization :) the question is, if the scheme we have agreed on for the smartdocs would be enough and if we want to change our current xmls - that would result in extra workload

Is there such a thing a as naming scheme for the Smart Docs, yet? We did start the discussion, but it never really got anywhere. This needs to happen first.

**Comment by jfaltenbacher** [comment]
I agree Jana. Can we try to come up with a scheme that would work for both? Otherwise we would get a bit stuck here

**Comment by cwickert** [comment]
Frankly speaking I only get half of Stefan's proposal. Can someone give me some example XML file names before we move on and think about smart docs?

**Comment by janajaeger** [comment]
> Frankly speaking I only get half of Stefan's proposal. Can someone give me some example XML file names before we move on and think about smart docs?

Which half remains unclear? The naming stuff is pretty much explained in his "does this spark joy" comment. And as for the Smart Docs, I'll provide the stuff we got so far after the SLE meeting today.

**Comment by janajaeger** [comment]
Naming scheme for Smart Docs as suggested by Frank can be found here: https://github.com/SUSE/doc-modular/tree/jjaeger_template-overhaul

**Comment by dariavladykina** [comment]
I would suggest not to rename or update the current files and images. We now have the naming scheme for Smart Docs so the question is whether to duplicate it in the style guide?

**Comment by fsundermeyer** [comment]
Seems to be resolved
